BUILDING.md

```diff
@@ -27,6 +27,8 @@ vcpkg install sail
 - `SAIL_ONLY_CODECS="a;b;c"` - Forcefully enable only the codecs specified in this ';'-separated list and disable the rest. If an enabled codec fails to find its dependencies, the configuration process fails. One can also specify not just individual codecs but codec groups by their priority like that: highest-priority;xbm. Default: empty list
 - `SAIL_OPENMP_SCHEDULE="dynamic"` - OpenMP scheduling algorithm. Default: dynamic
 
+- `BUILD_TESTING=ON|OFF` - Enable generation of tests using CTest. Default: `ON`
```

Comment:
Keep the list of the options sorted, i.e. move the BUILD_TESTING option to keep the list sorted